### PR TITLE
how to query in the querying section, correct default for select strategy, formatting

### PR DESCRIPTION
**Pull request by pjain1**

### CODE REVIEW

File: docs/content/configuration/indexing-service.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ its better to write a druid extension module with extending current worker selec
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`type`|`javascript`.|javascript|
+|`type`|`javascript`.|fillCapacity|
```
Comment (gianm) [inline docs/content/configuration/indexing-service.md]: hmm this seems very odd when placed under the "javascript" strategy. The default "type" for a javascript select strategy is "fillCapacity"?

IMO wording like "required; must be `javascript`" would make more sense.

File: docs/content/configuration/indexing-service.md [hunk 2]
```diff
@@ -192,7 +192,7 @@ An affinity config can be provided.
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`type`|`fillCapacityWithAffinity`.|fillCapacityWithAffinity|
+|`type`|`fillCapacityWithAffinity`.|fillCapacity|
```
Comment (gianm) [inline docs/content/configuration/indexing-service.md]: similar comment to the JS one.

File: docs/content/configuration/indexing-service.md (hunk 1 above)
Comment (pjain1) [inline docs/content/configuration/indexing-service.md]: I see what you are saying, let me make that change

Comment (pjain1) [inline docs/content/configuration/indexing-service.md]: done

File: docs/content/configuration/indexing-service.md (hunk 2 above)
Comment (pjain1) [inline docs/content/configuration/indexing-service.md]: done

**Comment by gianm** [comment]
:+1:

**Pull request merged**